DEVELOPMENT.md

```diff
@@ -48,6 +48,19 @@ You can use `docker stop medic-couchdb` to stop it and `docker start medic-couch
 
 Medic recommends you familiarise yourself with other Docker commands to make docker image and container management clearer.
 
+### CouchDB on Ubuntu
+
+While we recommend use Docker to install CouchDB for development, it is still possible to install CouchDB on bare metal in Ubuntu, but there's some caveats: 
+
+* For Ubuntu 18.04 and earlier, you need to specify in `apt` version to install with the `-V` flag.  For example, on a clean 18.04 install you would run:
+   ```bash
+   curl -sL https://couchdb.apache.org/repo/bintray-pubkey.asc | sudo apt-key add
+   echo "deb https://apache.bintray.com/couchdb-deb bionic main" | sudo tee -a /etc/apt/sources.list
+   apt update
+  apt install couchdb=2.3.1~bionic -V
+   ```
```

Comment:
Can we fix the whitespace alignment to be consistent?

Reply:
yuck!  Thanks for catching that. I'd copied the snippets from a site, tested them, and pasted them - hence the mish mash. fixed in 8884bab